app/containers/Network/index.js

```diff
@@ -174,14 +206,18 @@ const Network = () => {
     [data]
   );
 
-  if (loading) {
+  if (loading || loadingProfile) {
```

Comment:
remove `loadingProfile` this shouldn't block application, add the loading to the modal content